CTFd/utils/exports/__init__.py

```diff
@@ -165,16 +159,11 @@ def import_ctf(backup, erase=True):
 
     members = first + members
 
-    upgrade(revision=alembic_version)
```

Comment:
By removing this line, tables are created to the latest migration and imports will fail if they are from a previous version that had different data.

Reply:
Restored that line